pkg/event/event_test.go

```diff
@@ -35,8 +35,8 @@ func monkeyPatch() {
 				CurrentUtcOffsetValid: false,
 				Leap59:                false,
 				Leap61:                false,
-				TimeTraceable:         false,
-				FrequencyTraceable:    false,
+				TimeTraceable:         true,
+				FrequencyTraceable:    true,
 				PtpTimescale:          false,
 				TimeSource:            0,
```

Comment:
Is this the equivalent to TimeSource in ptp4l packets? Should 0x20 (GPS) for T-GM when GNSS connected.

Reply:
updated to GNSS when connected . and set to unknown when GNSS is lost